src/envoy/http/mixer/filter.cc

```diff
@@ -233,7 +227,12 @@ void Filter::log(const HeaderMap* request_headers,
 
     // Here Request is rejected by other filters, Mixer filter is not called.
     ::istio::control::http::Controller::PerRouteConfig config;
-    ReadPerRouteConfig(stream_info.routeEntry(), &config);
+    auto route_entry = stream_info.routeEntry();
```

Comment:
can you check for null to be defensive here?

Reply:
Ah, thanks a lot for catching this!